dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowProcessor.java

```diff
@@ -35,6 +35,8 @@
 	ContentletDependencies    contentletDependencies;
 	private final AtomicBoolean abort  = new AtomicBoolean(false);
 
+	private ConcurrentMap<String,Object> batchActionContext;
```

Comment:
I would just change the name since I think the context could be even valid on non -batch scenarios

Reply:
done